bin/travis-build.bash

```diff
@@ -15,29 +14,19 @@ echo "CKAN branch: $latest_ckan_release_branch"
 git checkout $latest_ckan_release_branch
 python setup.py develop
 pip install -r requirements.txt --allow-all-external
-pip install -r dev-requirements.txt --allow-all-external
 cd -
 
 echo "Creating the PostgreSQL user and database..."
 sudo -u postgres psql -c "CREATE USER ckan_default WITH PASSWORD 'pass';"
 sudo -u postgres psql -c 'CREATE DATABASE ckan_test WITH OWNER ckan_default;'
 
-echo "SOLR config..."
-# Solr is multicore for tests on ckan master, but it's easier to run tests on
-# Travis single-core. See https://github.com/ckan/ckan/issues/2972
-sed -i -e 's/solr_url.*/solr_url = http:\/\/127.0.0.1:8983\/solr/' ckan/test-core.ini
```

Comment:
I'd figure SOLR config would be needed for CKAN to run?

Reply:
Taking a look at the travis log of the current build (which is passing), i see that every package that is needed to run CKAN is installed. (line 10-17 in `travis-build.bash`). Here's the [log](https://travis-ci.org/CodeForAfricaLabs/ckanext-notify/builds/301478544?utm_source=github_status&utm_medium=notification)